torch/onnx/__init__.py

```diff
@@ -138,6 +137,36 @@
 producer_name = "pytorch"
 producer_version = _C_onnx.PRODUCER_VERSION
 
+_CUSTOM_ONNX_DECOMPS = {}
```

Comment:
I think user might want to see what they have registered and what not to avoid mess up the graph. Otherwise, it's hard to debug for them and for us.